x-pack/plugins/security_solution/cypress/e2e/detection_response/rule_management/maintenance_windows/maintenance_window_callout.cy.ts

```diff
@@ -8,51 +8,56 @@
 import { INTERNAL_ALERTING_API_MAINTENANCE_WINDOW_PATH } from '@kbn/alerting-plugin/common';
 import type { MaintenanceWindowCreateBody } from '@kbn/alerting-plugin/common';
 import type { AsApiContract } from '@kbn/alerting-plugin/server/routes/lib';
+import { tag } from '../../../../tags';
 import { cleanKibana } from '../../../../tasks/common';
 import { login, visit } from '../../../../tasks/login';
 import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../../urls/navigation';
 
-describe('Maintenance window callout on Rule Management page', () => {
-  let maintenanceWindowId = '';
-
-  before(() => {
-    cleanKibana();
-    login();
-
-    const body: AsApiContract<MaintenanceWindowCreateBody> = {
-      title: 'My maintenance window',
-      duration: 60000, // 1 minute
-      r_rule: {
-        dtstart: new Date().toISOString(),
-        tzid: 'Europe/Amsterdam',
-        freq: 0,
-        count: 1,
-      },
-    };
-
-    // Create a test maintenance window
-    cy.request({
-      method: 'POST',
-      url: INTERNAL_ALERTING_API_MAINTENANCE_WINDOW_PATH,
-      headers: { 'kbn-xsrf': 'cypress-creds' },
-      body,
-    }).then((response) => {
-      maintenanceWindowId = response.body.id;
+describe(
+  'Maintenance window callout on Rule Management page',
+  { tags: [tag.ESS, tag.SERVERLESS] },
```

Comment:
I'm curious if we're going to even have this feature in Serverless, because the maintenance windows management UI is in Stack Management in ESS. 🤔 

@approksiu or @peluja1012 do you know if in Serverless Security projects users will be able to create maintenance windows for rules?